### PR TITLE
Swap the meaning of 'a' and 'A' between aug and the absolute nominal

### PR DESCRIPTION
**Pull request by frostburn**
Add capital Greek Alpha as an alternative for augmented.

ref #174

### CODE REVIEW

**Comment by frostburn** [comment]
Reviewers assigned based on engagement with the issue. I'd very much like to hear what @arseniiv thinks about this before the PR is merged.

**Comment by frostburn** [comment]
Adding support for interval qualities longer than a single character deferred to #192 .

File: src/pythagorean.ts [hunk 1]
```diff
@@ -312,10 +327,10 @@ const IMPERFECT_QUALITY_SPECTRUM = [
   'n',
   'sM',
   'M',
-  'qA',
-  'sA',
+  'qa',
+  'sa',
   'Qa',
```
Comment (frostburn) [inline src/pythagorean.ts:332]: This spelling was actually a bug in `main`.

**Comment by arseniiv** [comment]
So,
- I’m not against aug and dim (but also I’m not a notationer).
- Greek Α might lead to copy-paste weirdness for users, but I guess it can be added.
- I glanced through the code and I’m pretty sure it’s not this, but if I missed it: if Alpha fills a gap in representation, it’s bad because code should be less ambiguous on read than UX, in my opinion.
- If the note is now `A` and augmented is now `a`, again that’s pretty much in balance as the previous state of things was, so it feels okay for al… beta that’s only just became beta. I feel very much underqualified for judgement but well my personal opinion is this.

I’ll look through the commits again and comment there is anything catches my eye.

**Comment by frostburn** [comment]
Good point about bad UX. How about `Å4` or something like that where it looks uppercase but has a diacritic for UX distinction?

**Comment by arseniiv** [comment]
That’s actually an interesting take!! IDK it feels esoteric but there are good sides like, especially for `Å` that it hints on pronunciation of *augmented* (vs. how the note A starts).

Comparing to other (singular) diacritics known to me (and common among Latin-scripted languages that are popular in the world), the ring looks quite prominent, cf. À Á Â Ã (tilde is quite prominent too) Ä Ā Ă Ą (interesting and likely found not much rarer than Å) Ȃ (esoteric) Ḁ (I’m joking) Ȧ Ạ (dots are way too small) Ɐ (joking again, but I can be retuned).

**Comment by frostburn** [comment]
I would go with `Â` because it looks like it goes up.

I only used `Å` because it's on the FI keyboard for biligual reasons. The issue with it that it stands for `O` not `A` and that `°` is comonly associated with *diminished* not *augmented* in music.

**Review by arseniiv — APPROVED** [review]
As far as I can see there are at least no typos in places you changed. Unfortunately I don’t feel like we could search for all possible occurrences of a/A used as augmented marker or the note (without finding just all a’s everywhere which is obviously overwhelming to look through). And uncaught typos in docs are simple to fix, whereas there aren’t that many places in the code that could be forgotten to be changed.

**Comment by arseniiv** [comment]
> The issue with it that it stands for `O` not `A` and that `°` is comonly associated with _diminished_ not _augmented_ in music.

Well the first is why I’m specifically okay with it because /ɔ/gmented is more o-like than a-like to me. The second is… yeah it’s a big point against.

**Comment by frostburn** [comment]
Swapped *Greek Alpha* for *Latin A with circumflex*.

@000masa000 feel free to merge if you're happy with these changes.

**Comment by frostburn** [comment]
Our resident HEJI consultant Richie Greene agrees that nominals should all be uppercase, so this is very likely to happen. I'll leave this PR open for Marc's comment, but will use the branch in an SW3 deployment.

**Comment by 000masa000** [comment]
Agree. Thanks!

**Review by 000masa000 — APPROVED** [review]

**Pull request merged**